resources/definitions/fdmprinter.def.json

```diff
@@ -1264,8 +1258,7 @@
                     "label": "Minimum Wall Line Width",
                     "description": "Width of the wall that will replace thin features (according to the Minimum Feature Size) of the model. If the Minimum Wall Line Width is thinner than the thickness of the feature, the wall will become as thick as the feature itself.",
                     "unit": "mm",
-                    "value": "wall_line_width_0 / 2",
-                    "default_value": "0.2",
```

Comment:
Don't remove the default_value. Those are used by the engine when using the command line interface. Each setting always needs a default_value, even though it's immediately overwritten by the "value". (Is this still the case? Are we still enforcing this?)

Reply:
> Is this still the case? Are we still enforcing this?

Yes. Cura sends the evaluated value for each setting to CuraEngine. If no value string is specified, the value evaluates to the default value.

In short: all settings need a default_value.

Reply:
OK, I will undo the default_value